apps/vs-code-designer/.eslintrc.json

```diff
@@ -0,0 +1,39 @@
+{
+    "extends": [
+        "../../.eslintrc.json"
+    ],
+    "ignorePatterns": [
+        "!**/*",
+        "**/node_modules/**",
+        "**/build/**",
+        "**/__test__/**",
+        "**/__tests__/**"
+    ],
+    "overrides": [
+        {
+            "files": [
+                "*.ts",
+                "*.tsx",
+                "*.js",
+                "*.jsx"
+            ],
+            "rules": {}
+        },
+        {
+            "files": [
+                "*.ts",
+                "*.tsx"
+            ],
+            "rules": {
+                "react-hooks/rules-of-hooks": "off"
```

Comment:
I wouldn't advise ever turning this off, but disable it where it's not needed instead

Reply:
The `vs-code-designer` doesn't host any React code. It's just giving errors in a function that is called `useBinariesDependencies`. Eslint thinks that its a hook and its not in a react component or in a .tsx file. Thats why I decided to remove the whole rule in that project